bin/src/hickory-dns.rs

```diff
@@ -44,6 +44,16 @@ use std::{
 };
 
 use clap::Parser;
+
+#[cfg(feature = "__prometheus")]
+use metrics_exporter_prometheus::PrometheusBuilder;
+#[cfg(feature = "__prometheus")]
+use metrics_tracing_context::MetricsLayer;
+
+use hickory_dns::Config;
+#[cfg(feature = "__tls")]
+use hickory_dns::TlsCertConfig;
+use hickory_server::{authority::Catalog, server::ServerFuture};
```

Comment:
Please leave the `hickory_dns` imports in place.

Reply:
Reverted.